docs/howto/openstack/neutron/vpnaas.md

```diff
@@ -0,0 +1,786 @@
+    | 5f44be31-0588-4f33-883d- | vpn-conn-to-kna1 | 203.0.113.101 | psk                      | ACTIVE |
+    | 9e2d97be55e1             |                  |               |                          |        |
+    +--------------------------+------------------+---------------+--------------------------+--------+
+    ```
+        
+    That would be `vpn-conn-to-kna1`, and according to the command 
+    output above, it is active. To disable it, type the following:
+        
+    ```bash
+    openstack vpn ipsec site connection set --disable vpn-conn-to-kna1
+    ```
+        
+    Check if it is really disabled:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+        
+    It looks like it is disabled --- or `DOWN`.
+        
+    > You will probably have to wait several seconds before seeing a 
+    status change. If you are impatient, try something like this:
+    ```bash
+    watch "openstack vpn ipsec site connection show vpn-conn-to-kna1 -c Status"
+    ```
+    Hit CTRL+C as soon as you see the status change you expect.
+        
+    Now, get on the right side of the connection (region `kna1`), 
+    optionally look for the name of the VPN connection to the left (in our 
+    example, that would be `vpn-conn-to-fra1`), and check its status:
+        
+    ```bash
+    openstack vpn ipsec site connection show vpn-conn-to-fra1 -c Status
+    ```
+        
+    ```plain
+    +--------+-------+
+    | Field  | Value |
+    +--------+-------+
+    | Status | DOWN  |
+    +--------+-------+
+    ```
+    
+    It turns out that this direction of the connection is also disabled.
+
+To test that a previously enabled site-to-site connection is now disabled,
+select a server from one region and try to ping a server in another region.
+Suppose, for example, that a now-disabled connection involved regions
+`fra1` and `kna1`, and that we have servers `server-fra1` (in `fra1`) and
+`server-kna1` (in `kna1`). With the VPN connection between the two regions
+now disabled, pinging should not be possible:
+
+```console
+ubuntu@server-fra1:~$ ping -c 3 10.15.20.148
+PING 10.15.20.148 (10.15.20.148) 56(84) bytes of data.
+
+--- 10.15.20.148 ping statistics ---
+3 packets transmitted, 0 received, 100% packet loss, time 2055ms
+```
+
+```console
+ubuntu@server-kna1:~$ ping -c 3 10.15.25.58
+PING 10.15.25.58 (10.15.25.58) 56(84) bytes of data.
+
+--- 10.15.25.58 ping statistics ---
+3 packets transmitted, 0 received, 100% packet loss, time 2050ms
+```
+
+## Enabling an inactive connection
+
+You can easily enable an inactive site-to-site VPN connection.
+
+=== "{{gui}}"
+    Currently, there is no way to enable an inactive connection from 
+    the {{gui}}. If you want to disable an inactive connection, please use 
```

Comment:
Apologies, just caught this now in final review: `s/disable/enable/` (or maybe use "re-enable").

Reply:
Aha, good catch! I say we go with "re-enable".